stdlib/public/core/ExistentialCollection.swift.gyb

```diff
@@ -672,22 +570,19 @@ extension _ClosureBasedSequence: Sequence {
 /// An instance of `AnySequence` forwards its operations to an underlying base
 /// sequence having the same `Element` type, hiding the specifics of the
 /// underlying sequence.
-//@usableFromInline
+//
 @_fixed_layout
 public struct AnySequence<Element> {
```

Comment:
I'd be curious about the benchmark results for un-fixed-layout'ing these, in a separate PR